Libraries/Components/ScrollView/ScrollView.js

```diff
@@ -676,7 +688,7 @@ const ScrollView = createReactClass({
    * @platform ios
    */
   flashScrollIndicators: function() {
-    this.getScrollResponder().scrollResponderFlashScrollIndicators();
+    this._scrollResponder.scrollResponderFlashScrollIndicators();
```

Comment:
I think we should just change `getScrollResponder()` to return `this._scrollResponder`.

Reply:
Not sure about that. According to the comments `getScrollResponder()` is used to e.g. fire `scrollTo()` which is not available in `ScrollResponder.Mixin`. Unless they're wrong (there's no actual usage of `getScrollResponder()` in this repo, so it may relate to internal FB callsites)